packages/discovery-react-components/src/components/SearchResults/components/Result/Result.tsx

```diff
@@ -113,7 +113,12 @@ export const Result: React.FunctionComponent<ResultProps> = ({
   if (result) {
     documentId = result.document_id;
   }
-  const title = get(result, resultTitleField);
+
+  let title = get(result, resultTitleField);
+  if (title && title.length > 100) {
```

Comment:
is there a reason we decided on `100`? also would be good to enable a hover of the title to show the entire title if we are clipping it

Reply:
100 was an arbitrary choice that I felt would be able to contain most customers title choices. I'm open to feedback for a different value and agree that having a hover to display the full title would make for a better experience here